webapp/graphite/carbonlink.py

```diff
@@ -97,6 +97,12 @@ def precheck(self, metric, timestamp=None):
     log.cache("CarbonLink cache-query-precheck request for %s" % (metric))
     return results["exists"]
 
+  def expand_query(self, metric):
```

Comment:
I'd suggest calling this some closer to the carbolink query type, like expand_wildcards.